02-peer-protocol.md

```diff
@@ -100,6 +101,14 @@ desire to set up a new channel.
    * [33:first-per-commitment-point]
 
 
+The `chain-hash` value denotes the exact blockchain the opened channel will
```

Comment:
Too blockchain-specific.  We should make the language more flexible, eg:

The `chain-hash` value denotes the exact blockchain the opened channel will
reside within. This is usually the _genesis hash_ of the respective blockchain.
The existence of the `chain-hash` allows nodes to open channel
across many distinct blockchains as well as have channels within multiple
blockchains opened to the same peer (if they support the target chains).